modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/display/context/logic/PanelCategoryHelper.java

```diff
@@ -171,6 +172,13 @@ protected boolean hasPortlet(
 		return false;
 	}
 
+	public List<PanelCategory> getPanelCategories(
```

Comment:
Why is this method needed? You have access to `PanelCategoryRegistryUtil` where this method is called.

Reply:
*Util/Helper classes with static method usually are anti-patterns and it makes harder to test. It's a good idea to avoid them. Since there is this `PanelCategoryHelper` class as well( which does not contains static methods all over the place), it makes easier to encapsulate the *Util call inside of it.